djangocms_blog/models.py

```diff
@@ -326,6 +326,10 @@ def get_posts(self, request):
             posts = posts.filter(categories__in=list(self.categories.all()))
         return posts.distinct()[:self.latest_posts]
 
+    def get_plugin_name(self):
+        from cms.plugin_pool import plugin_pool
+        return plugin_pool.get_plugin(self.plugin_type).name.format(self.app_config.namespace)
```

Comment:
``self.app_config.object_name`` is the right attribute to use here (see https://github.com/nephila/djangocms-blog/blob/develop/djangocms_blog/cms_appconfig.py#L21)